app/networkapi/settings.py

```diff
@@ -130,8 +131,8 @@
                 'mezzanine.template.loader_tags',
             ],
             'libraries': {
-                'features_adminsortable_tags': 'networkapi.features.templatetags'
-                                               '.features_adminsortable_tags_custom'
+                'adminsortable_tags':
+                    'networkapi.utility.templatetags.adminsortable_tags_custom'
```

Comment:
I moved the templatetags to utility because realistically, they are shared between all the apps and, since the templatetag names are not something we came up with (adminsortable named them and uses them throughout their templates), we cannot have the same name for templatetags across multiple apps.